labs/sled-state-inspector/src/main.rs

```diff
@@ -4,7 +4,7 @@ use atty::Stream;
 use clap::{Arg, ArgMatches, Command as Argparse};
 use futures::executor::block_on;
 use matrix_sdk_base::RoomInfo;
-use matrix_sdk_sled::StateStore;
+use matrix_sdk_sled::{SledStateStoreBuilder, StateStore};
```

Comment:
I would prefer if we didn't have to import builder types separately. Can we have `StateStore::builder()` instead / use it if we already have it?